PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/referencebook/AzureSdkArtifactGroupPanel.java

```diff
@@ -109,4 +125,41 @@ private void createUIComponents() {
         this.toolbar.setForceMinimumSize(true);
         this.toolbar.setTargetComponent(this.viewer);
     }
+
+    /**
+     * referred com.intellij.application.options.schemes.AbstractSchemesPanel.ShowSchemesActionsListAction
+     *
+     * @see com.intellij.application.options.schemes.AbstractSchemesPanel
+     */
+    private static class ShowDependencyTypeListAction extends DefaultActionGroup {
+        private String selected;
+
+        private ShowDependencyTypeListAction(Consumer<? super String> onTypeSelected) {
+            super();
+            setPopup(true);
+            final Consumer<String> onSelected = (type) -> {
```

Comment:
not good to use final Consumer<String> onSelected = (type) here

Reply:
why?